packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -398,14 +398,18 @@ The following code shows how different attributes provided by 'Login With Amazon
 user pool attributes.
 
 ```ts
-new UserPoolIdentityProviderAmazon(stack, 'Amazon', {
-  // ...
+const userpool = new cognito.UserPool(this, 'Pool');
+
+new cognito.UserPoolIdentityProviderAmazon(this, 'Amazon', {
+  clientId: 'amzn-client-id',
+  clientSecret: 'amzn-client-secret',
+  userPool: userpool,
   attributeMapping: {
```

Comment:
Not a fan of this change either. Now the focus is diffused and doesn't focus on just the `attributeMapping` property which this section is talking about.

How about defining the whole set of properties in the rosetta fixture and doing something like

```ts
new UserPoolIdentityProviderAmazon(stack, 'Amazon', {
  ...otherIdentityProviderAttrs, // this is defined in the fixture
  attributeMapping: {
  }
});
```